Source/Xamarin/Prism.Autofac.Forms/Navigation/AutofacPageNavigationService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Autofac;
+using Prism.Common;
+using Prism.Logging;
+using Prism.Navigation;
+using Xamarin.Forms;
+
+namespace Prism.Autofac.Navigation
+{
+    public class AutofacPageNavigationService : PageNavigationService
+    {
+        readonly IContainer _container;
+
+        public AutofacPageNavigationService(IContainer container, IApplicationProvider applicationProvider, ILoggerFacade logger)
+            : base(applicationProvider, logger)
+        {
+            _container = container;
+        }
+
+        protected override Page CreatePage(string name)
+        {
+            if (!_container.IsRegisteredWithName<Page>(name))
+                throw new NullReferenceException($"The requested page 'missing' has not been registered.");
```

Comment:
I think you were going for something more like:

`throw new NullReferenceException($"The requested page '{name}' has not been registered.");`